config/config_test.go

```diff
@@ -80,7 +80,12 @@ func TestLoadConfig(t *testing.T) {
 								MaxExecutionTime:     Duration(time.Minute),
 							},
 						},
-						HeartBeatInterval: Duration(time.Minute),
```

Comment:
I would leave tests for `HeartBeatInterval` without `HeartBeat` section in order to make sure all the existing configs won't break after the upgrade.

Reply:
Done